src/tests/ftest/util/server_utils.py

```diff
@@ -230,6 +230,20 @@ def clean_files(self, verbose=True):
             verbose (bool, optional): display clean commands. Defaults to True.
         """
         clean_commands = []
+
+        if self.manager.job.yaml.control_metadata.device:
```

Comment:
Nothing in ftest is yet using `device`, right?

Reply:
Right. This is for the vos_on_blob branch, where we need at least the external directory for control metadata to be working soon. The plan is to eventually start testing with the control metadata device as well.